go/kernel/linux/eventfd.go

```diff
@@ -1,25 +1,27 @@
 package linux
 
 import (
+	"fmt"
+	"github.com/lunixbochs/usercorn/go/kernel/posix"
 	"syscall"
 )
 
 func (k *LinuxKernel) Eventfd2(initval, flags uint) uint64 {
 	var a3 uintptr
-	r1, _, err := syscall.Syscall(syscall.SYS_EVENTFD2, uintptr(initval), 
-uintptr(flags), a3)
+	r1, _, err := syscall.Syscall(syscall.SYS_EVENTFD2, uintptr(&initval), uintptr(&flags), a3)
+	fmt.Println(r1)
```

Comment:
I'm not going to merge printf debugging information in a syscall, remember you can -strace to see the input/output of a syscall.